### PR TITLE
Feature/get with scraper

### PR DESCRIPTION
**Pull request by KLPainter**
Returns records from mcso, filtering out the ones that already have an associated booking.

### CODE REVIEW

File: server/src/services/__mocks__/scraper.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+const mockMscoRecords = [
```
Comment (acrankin) [inline server/src/services/__mocks__/scraper.js]: msco should be mcso

File: server/src/routes/api/orgs.js [hunk 2]
```diff
@@ -21,4 +22,22 @@ export default Router()
...
+
+    const promises = [
+      Booking.find(query)
+              .then(bookings => bookings.map(booking => booking.swisId)),
+      getMcsoRecords()
+    ];
+
+    Promise.all(promises)
+      .then(([swisIds, mcsoBookings]) => {
+        return mcsoBookings.filter(msco => !swisIds.includes(msco.swisId))
```
Comment (acrankin) [inline server/src/routes/api/orgs.js]: msco should be mcso

File: server/src/services/__mocks__/scraper.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  mcsoHair: 'Black',
+  mcsoEyes: 'Brown',
+  mcsoArrestingAgency: 'Portland Police, North Precinct',
+  mcsoBookingDate: '12/08/2018 04:03 AM',
+  mcsoAssignedFacility: 'MCIJ',
+  mcsoProjectedReleaseDate: 'Unknown' }
+];
+
+export const getMcsoRecords = () => {
+  return Promise.resolve(mockMscoRecords);
```
Comment (acrankin) [inline server/src/services/__mocks__/scraper.js]: one last one, sorry!

File: server/src/services/__mocks__/scraper.js (hunk 1 above)
Comment (KLPainter) [reply server/src/services/__mocks__/scraper.js]: Thanks, Eagle Eye.  👍

**Pull request merged**